pkg/apis/kudo/v1beta1/operatorversion_types.go

```diff
@@ -71,6 +71,20 @@ type Plan struct {
 	Phases []Phase `json:"phases"`
 }
 
+// ParameterValueType specifies the type of a parameter value.
+type ParameterValueType string
```

Comment:
couldn't this be abbr to `ParameterType`?  value seems redundant... it is the only thing that can be meant right?
```suggestion
type ParameterType string
```